### ISSUE TITLE
Walking into terrirtory

### ISSUE THREAD

**Issue by TMNetwork** [drtshock/HCFProblems#35]
Can you make it where when we walk into a territory that we can config that msg.

Example:
&eYou are now entering: &b{faction name}

and when they leave
&eYou are now leaving: &b{faction name}

**Comment by nsporillo** [comment]
Duplicate of https://github.com/drtshock/HCFProblems/issues/22

This feature is going to be implemented, just not until atleast monday.
